### PR TITLE
Container kubernetes centos

### PR DESCRIPTION
**Pull request by mohammedzee1000**

### CODE REVIEW

File: containers/kubernetes/install.sh [hunk 1]
```diff
@@ -35,4 +40,6 @@ chmod 775 /usr/share/cockpit/shell
 chmod 775 /usr/share/cockpit/kubernetes
 
 # Move kubernetes index file away so we only link it when we want it
-mv /usr/share/cockpit/kubernetes/index.html.gz /usr/share/cockpit/kubernetes/original-index.gz
+if [ -z "$CENTOS" ]; then
```
Comment (petervo) [inline containers/kubernetes/install.sh]: This is needed even for centos.

File: containers/kubernetes/install.sh [hunk 2]
```diff
@@ -3,14 +3,19 @@
...
-/container/scripts/install-rpms.sh -a noarch --nodeps cockpit-shell-
-/container/scripts/install-rpms.sh cockpit-ws-
-/container/scripts/install-rpms.sh --nodeps cockpit-kubernetes-
+if [ -z "$CENTOS" ]; then
+	/container/scripts/install-rpms.sh --nodeps cockpit-bridge-
+	/container/scripts/install-rpms.sh -a noarch --nodeps cockpit-shell-
+	/container/scripts/install-rpms.sh cockpit-ws-
+	/container/scripts/install-rpms.sh --nodeps cockpit-kubernetes-
+else
+	INSTALL_PKGS="cockpit cockpit-bridge cockpit-shell cockpit-ws cockpit-kubernetes"
```
Comment (petervo) [inline containers/kubernetes/install.sh]: cockpit-bridge and cockpit-kubernetes need to be installed without installing any of their dependencies.

**Comment by mohammedzee1000** [comment]
Thanks. I will work on getting your requested changes in. 
As far as documentation about the centos container pipeline, You can look at the the centos wiki page of the same https://wiki.centos.org/ContainerPipeline?action=show and also https://github.com/CentOS/container-pipeline-service/blob/master/README.md for how it works. 
@kbsingh and @bamachrn Do you wish to add?

**Comment by stefwalter** [comment]
@petervo Could you comment on what still needs to be done here?

**Comment by petervo** [comment]
My comments are still on the review, we should not install deps and the index setup still needs to happen on centos

**Comment by petervo** [comment]
replaced by #5539

**Pull request closed without merging**